gridstatus/ercot.py

```diff
@@ -1773,6 +1777,41 @@ def _make_bid_curve(df):
 
         return self.parse_doc(df, verbose=verbose)
 
+    # @support_date_range("DAY_START") ??
+    def get_sced_system_lambda(self, date, verbose=False):
+        """Get System lambda of each successful SCED
+
+        5 Minute Publish Interval
+
+        https://github.com/kmax12/gridstatus/issues/269
+
+        Arguments:
+            date (str, datetime): date to get data for
+            verbose (bool, optional): print verbose output. Defaults to False.
+
+        Returns:
+            pandas.DataFrame: A DataFrame
+
+        """
+
+        doc = self._get_document(
```

Comment:
let's make this function return a full day of data if `date != 'latest'`

in that case where `date != latest`, we can just pass the date along to use self._get_documents instead and it will return a list of documents for that day

you can then parse and concatenate them

Reply:
Is speed an issue you solve for?

As of now, fetching a whole day takes awhile.

I can speed it up if by batch fetching, but only want to do that if necessary

Reply:
i think supporting a whole day and latest is enough for now. in theory it be nice to be able to specify a finer granularity, but that would require some reworking to get_documents. i would totally welcome it, but maybe out of scope

Reply:
Just went the easy route on this one